ansible_collections/arista/avd/roles/eos_validate_state/ANTA-Preview.md

```diff
@@ -222,6 +227,21 @@ skipped_tests:
       - VerifyEnvironmentCooling
 ```
 
+## Custom ANTA catalog
+
+You can now provide custom ANTA catalogs to the AVD `eos_validate_state` role. By default, AVD will search for catalog YAML files in the `custom_anta_catalogs` directory and incorporate these tests into the existing dynamically created catalog from AVD. The custom catalog files must be named as follows:
+
+- `<hostname>.yml` or `<hostname>.yaml`
+- `<group>.yml` or `<group>.yaml`
+
+When specifying a group, it must be a group from the Ansible inventory. The custom tests will then be added to all devices that are part of this group. You can also use the `all` group to target all the devices in your inventory. The directory where the custom catalogs are stored can be changed with the `custom_anta_catalogs_dir` variable.
+
+!!! warning
+    The `skipped_tests` variable will ONLY skip the dynamically generated tests from the AVD validate state role. It will **not** skip tests added from custom catalogs.
+
+!!! info
+    The final catalog will be validated by ANTA before running the tests on your network. Duplicate tests will be automatically removed.
```

Comment:
is there any caveat on merging to know - can I overwrite a generated test?

Reply:
Explained better in c490e182faf21a5c7f05d8c9fc38f181a875f708.